posts/2016-10-10-serverless-v1.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Serverless V1 release
+date: 2016-05-25
+layout: Post
+---
+Goals of the Blogpost:
+
+* Proud to release this new version
+* Lot of work over the last months to reimplement Serverless with goal of
+  * Easier onboarding
+  * Full Cloudformation Support
+  * Easier time for users to contribute
+  * Tight control over the whole infrastructure you're deploying
+  * Preparation for more complex features for a more complex infrastructure
+* Set out to understand what small, medium and large teams want to use Serverless for
+* Build a stronger Contributor community
+* Build basic platform to manage large serverless infrastructure
+* Prepare for multi-provider Support in the future
+* Key features
+  * Easy event support for many different AWS resources
+  * Full Cloudformation support so you can define custom resources
+  * Better APIG Support
+  * easily extensible plugin System
+  * Powerful variable system
+  * Lots of convenience functions to build your infrastructure
+  * Better documentation to find things easier
+  * Improved Code quality and test coverage so we can move much faster in the future
+* Call to Action to contribute
+* Future of Serverless Framework
+  * Serverless wants to be the Platform to build large scale event driven micro-service applications
+  * Serverless V1 is first step there
+    * Allows to easily build, configure,
+  * Service Discovery
+  * Service Composition
+  * Service Communication
+
+
+# Serverless 1.0 Release
+
+Very happy to announce final release of Serverless 1.0. We worked very hard on making Serverless an amazing tool over the last year and this is the culmination of many conversations with many teams.
+
+Serverless should be best tool to manage FaaS infrastructure for any team. From small to large to enterprise we see so many jump on FaaS and Serverless. They all get the advantage of focusing fully on their business code and not having to do operations for many things in their infrastructure.
+
+To get started all you really need to do is run the following commands (or watch this video):
+
+```
```

Comment:
Digging the outline =)

Just FYI/Quick note on code snippets. 

Use the language with the code block to get syntax highlighting just like on github. aka `bash` here

"```bash
serverless rocketlaunch -target moon
```
"